docs/general/server/media/music.md

```diff
@@ -5,94 +5,110 @@ title: Music
 
 # Music
 
-The most common organization scheme for music is separation by artist and then album.
+You should organize albums into folders, with one folder containing one and only one album. Jellyfin does not care how you organize albums together, as long as each album is contained within one folder. Filenames generally do not matter since the info will be scraped from the embedded metadata of the tracks. If no other metadata was found, Jellyfin uses the file names as track titles.
```

Comment:
```suggestion
Albums are organized in folders, with one folder containing one and only one album. Jellyfin does not care how you organize albums together, as long as each album is contained within one folder. Filenames generally do not matter since the info will be scraped from the embedded metadata of the tracks. If no other metadata was found, Jellyfin uses the file names as track titles.
```